src/components/PhotoAlbum.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Photos from './Photos';
+
+function PhotoAlbum ({ photos }) {
+  return(
+    <div>
+    <h1>Photo Album</h1>
+    <Photos urls={photos} />
```

Comment:
```suggestion
    <Photos photos={photos} />
```
This is the next part where it needs to be "photos" to continue getting passed down (at least for how you have it set up right now).